### ISSUE TITLE
Uncaught TypeError: paneContainerElement.nearestPaneInDirection is not a function

### ISSUE THREAD

**Issue by angelocordon** [joslarson/atom-simple-panes#1]
1. Using Atom Beta, when focused on a pane, pressing `ctrl+k` and then left or arrow key

**Atom**: 1.19.0-beta1 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.5
**Thrown From**: [simple-panes](https://github.com/joslarson/atom-simple-panes) package 1.1.0

### Stack Trace

Uncaught TypeError: paneContainerElement.nearestPaneInDirection is not a function

```
At /Users/angelocordon/.atom/packages/simple-panes/index.js:7

TypeError: paneContainerElement.nearestPaneInDirection is not a function
    at simplePanesMoveItem (/packages/simple-panes/index.js:7:57)
    at HTMLElement.atom.commands.add (/packages/simple-panes/index.js:32:5)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:265:35)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:3:65
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
  2x -9:06.3.0 core:select-up (input.hidden-input)
     -9:05.8.0 editor:select-to-first-character-of-line (input.hidden-input)
     -9:04.9.0 core:move-left (input.hidden-input)
     -9:04.7.0 core:move-right (input.hidden-input)
     -9:04.5.0 core:move-left (input.hidden-input)
     -9:04.4.0 core:move-right (input.hidden-input)
     -9:04.3.0 core:move-left (input.hidden-input)
     -9:04.2.0 core:move-right (input.hidden-input)
  2x -9:04.1.0 core:move-left (input.hidden-input)
     -0:14.8.0 core:select-right (input.hidden-input)
     -0:13.5.0 core:move-down (input.hidden-input)
     -0:11.1.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
  2x -0:07.7.0 core:backspace (input.hidden-input)
     -0:07.0 core:confirm (input.hidden-input)
     -0:05.8.0 tree-view:toggle (input.hidden-input)
     -0:04.3.0 simple-panes:move-item-right-of-pane (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
busy-signal 1.4.3 
emmet 2.4.3 
intentions 1.1.2 
language-vue 0.23.0 
line-breaker 0.4.0 
linter 2.2.0 
linter-eslint 8.2.1 
linter-rubocop 2.1.1 
linter-ui-default 1.6.2 
platformio-ide-terminal 2.5.1 
simple-panes 1.1.0 
```

**Comment by angelocordon** [comment]
hey @joslarson, thanks so much for this awesome plugin for Atom - I've been loving it and have been using it for a long time. Recently I've been getting this error and it's out of the blue as I don't think there were any updates with this plugin for a while. 

Let me know if I've filled out this issue log correctly (was just following instructions from Atom), or if you need more info, or even where the issue may be.

Thanks in advance for your help!

**Comment by joslarson** [comment]
@angelocordon thanks for taking the time to bring this to my attention :) That's everything I need. I expected this would happen eventually as this package is using an undocumented function. I found the offending commit here: https://github.com/atom/atom/commit/81b05023dbb6a3048a17ea0e1810f9af687afdc3

I've got it working locally. I just need to make sure I've got the logic right to keep it working in older versions of Atom as well.

Thanks again.
